src/chat/participant.ts

```diff
@@ -275,23 +275,7 @@ function filterContextHistory(
   if (filteredHistory.length === 0) {
     return [];
   }
-  const messages: LanguageModelChatMessage[] = [];
-  for (const turn of filteredHistory) {
-    // don't re-use previous prompts since the model may misinterpret them as part of the current prompt
-    if (turn instanceof ChatRequestTurn) {
-      if (turn.participant === PARTICIPANT_ID) {
-        messages.push(userMessage(turn.prompt));
-      }
-      continue;
-    }
-    if (turn instanceof ChatResponseTurn) {
-      // responses from the participant:
-      if (turn.response instanceof ChatResponseMarkdownPart) {
-        messages.push(participantMessage(turn.response.value.value));
-      }
-    }
-  }
 
-  logger.debug("filtered messages for historic context:", messages);
```

Comment:
The message formatting was complicating this function too much, so it got its own `summarizeChatHistory()` helper function in `src/chat/summarizers/`

Reply:
Handled in https://github.com/confluentinc/vscode/pull/1740 instead of this branch